### PR TITLE
Added janw-me to the Codeowners for the PHP FSE folder.

### PR DESCRIPTION
**Pull request by janw-me**
I'd like to get the updates for this folders.
I apologize if this is the wrong approach.

### CODE REVIEW

File: .github/CODEOWNERS
Comment (priethor) [inline .github/CODEOWNERS]: Should `@spacedmonkey` and `@timothybjacbos` also be added as code owners? As far as I understand from your request, you should be the only one added, isn't that so?

**Review by priethor — CHANGES_REQUESTED** [review]
Hi @janw-me !

This would be one way of getting notifications, although you would also get added as a reviewer on every PR that affects files in that folder.

As an alternative, if you are only interested in getting notifications once changes to that folder happen in `trunk`, there's also the possibility of setting up an RSS feed with the following URL:
`https://github.com/WordPress/gutenberg/commits/trunk/lib/full-site-editing.atom`

However, in case you do want to be added to the code owners, I left a review to address the changes.

**Comment by janw-me** [comment]
Hi @priethor

Thank you for the atom feed. That's quite usefull.
But if it's allowed I'd like to be pinged before it's merged.

File: .github/CODEOWNERS
Comment (janw-me) [reply .github/CODEOWNERS]: Yes, I only should be enough. I can't remember why I added the others :thinking:
I pushed a commit to fix this.

**Review by priethor — APPROVED** [review]

**Pull request merged**
